### PR TITLE
Enable/disable delivery methods checkboxes on the request form

### PR DESCRIPTION
**Pull request by bogdan-zaharia**
DES-155: https://hackney.atlassian.net/jira/software/projects/DES/boards/45?selectedIssue=DES-155

Add the option to check/uncheck the checkboxes for delivery methods on the new evidence request form

### CODE REVIEW

File: src/components/Checkbox.tsx [hunk 1]
```diff
@@ -20,6 +23,8 @@ export interface Props {
   label: string;
   name: string;
   value?: string;
+  disabled?: boolean;
+  checked?: boolean;
```
Comment (neilmendum) [inline src/components/Checkbox.tsx:27]: I think we can remove this property because it isn't used

File: src/components/Checkbox.tsx [hunk 2]
```diff
@@ -8,6 +8,9 @@ const Field = (props: Props): JSX.Element => (
       name={props.name}
       id={props.name}
       value={props.value}
+      disabled={props.disabled}
+      //checked={props.value != ''}
+      //checked={!props.disabled && props.value != ''}
```
Comment (neilmendum) [inline src/components/Checkbox.tsx:13]: I think we can remove these comments

File: src/components/NewRequestForm.tsx [hunk 3]
```diff
@@ -60,13 +74,34 @@ const NewRequestForm = ({
     [setRequest, request, onSubmit, setSubmitError]
   );
 
+  const buildEvidenceRequestRequest = (values: EvidenceRequestForm) => {
+    const deliveryMethods: string[] = [];
+    console.log('VALUES', values);
```
Comment (neilmendum) [inline src/components/NewRequestForm.tsx:79]: We can remove this log

**Review by neilmendum — COMMENTED** [review]
Added a few comments

File: src/components/Checkbox.tsx (hunk 1 above)
Comment (bogdan-zaharia) [reply src/components/Checkbox.tsx:27]: I think `disabled` is still being used, but `checked` can be removed

**Review by neilmendum — APPROVED** [review]
LGTM

**Pull request merged**